src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -0,0 +1,64 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/get_output_element.hpp"
+#include "ngraph/op/topk.hpp"
+#include "ngraph/type/element_type.hpp"
+#include "topk.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector topk(const Node& node)
+                {
+                    auto data = node.get_ng_inputs().at(0);
+                    std::int64_t axis{node.get_attribute_value<std::int64_t>("axis", -1)};
+                    std::int64_t k{node.get_attribute_value<std::int64_t>("k")};
+
+                    if (axis < 0)
```

Comment:
You may use: `convert_negative_axis` https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/frontend/onnx_import/utils/common.hpp#L149 here.

Reply:
I agree that I should check if axis > "number of axis", but this function take the last one.
I will add an assert to check it.

Reply:
Truncate provided axis means the same as take the last one because axis is used to select axis.